javascript/README.md

```diff
@@ -5,12 +5,28 @@ If you are not familiar with linters and Stickler, read [root level README](../R
 
 Please do the following **steps in this order**:
 
-1. Install stickler-ci https://github.com/apps/stickler-ci
-2. Enable stickler in your repo. You can do it [here](https://stickler-ci.com/).
-3. In first commit of your feature branch add a copy of [.stickler.yml](./.stickler.yml) and [eslint.config](./eslint.config) .
-    - **Remember** to use both files linked above
+1. Install eslint
```

Comment:
@dipto0321 I agree we need to add it but why not here: https://github.com/microverseinc/linters-config/pull/16/files#diff-41be41fc0c21938ff1ba8b2fdfa6291cR45 ?

Reply:
Hi @nidalaa ,

I didn't understand what do you mean? Do you mean to say I should add it in **Troubleshooting** section?

Reply:
Yes @dipto0321 :)

Reasons:
- Stickler is going to work even without linter configured locally
- in Troubleshooting we provide instructions on how to work with linter in local env, so this line of code https://github.com/microverseinc/linters-config/pull/16/files#diff-41be41fc0c21938ff1ba8b2fdfa6291cR45 looks like a perfect place for those instructions

Reply:
@dipto0321 can you also repeat it in newly added react&reduc README ?

Reply:
Hi @nidalaa ,

Yes, it's true that stickler is going to work without linter configured locally. But in this case, we need to create different linter config files according to the projects. In an example, javascript first project is a common js related project. So it's linter config is different than the current one. Not only that it also different for webpack related project. Then if somebody uses babel in webpack than it will be again different. It also needs a different configuration for the javascript first test project. Because that project is the only console base I mean we need to write some methods and then need to write some test. There is nothing for browser. So for that project my linting configuration will also be changed. Now we also know that when it comes to react project than it's again changed. That's why I highly recommend adding linter install point before troubleshooting. Not inside troubleshooting. And I think it also a good practice about how to use a linter. Student can learn which linter configuration is suitable for there project.
Thanks 🙏

Reply:
@dipto0321 those config files are predefined because we want all students to follow the same linters rules.
So, instead of giving instructions on what to choose during `npx eslint --init`, I'd prefer to generate one `.eslintrc.js` and share it in this repo.


Can I have additional questions:
- why the current `./eslint.config` is worse than `eslintrc.js`? is it missing something? does it include too many rules?
- can you think about config option that serves for the maximum number of projects? 

I'd love to have as little different config files in JS section as possible.
MAybe the ditinction webpack - not webpack would be enough?